### ISSUE TITLE
Windows Terminal - Use Arch Icon instead of the generic Linux icon

### ISSUE THREAD

**Issue by julianxhokaxhiu** [yuk7/wsldl#176]
Hello,

I am approaching you as an outcome of a discussion I had on https://github.com/sileshn/ArchWSL2/issues/13

TLDR; I am curious to know if it's possible to embed the native Arch Linux logo when adding the entry on Windows Terminal. Accordingly to the ArchWSL2 author it seems to be a wsldl thing, however exploring the code I am not able to find any mention of Arch, and it's yet unclear to me how the `Arch.exe` is being built, otherwise I would have submitted a PR with this change instead of asking for it.

Any help on accomplishing this little detail is appreciated.

Thank you in advance,
Julian

**Comment by yuk7** [comment]
Yes, ArchWSL2 uses wsldl, and as he says, if you want the feature to set the icon in Windows Terminal, we will need to change the wsldl implementation

**Comment by yuk7** [comment]
Thank you, I read the issue.
Yes, ArchWSL2 uses wsldl, and as he says, if you want the feature to set the icon in Windows Terminal, we will need to change the wsldl implementation basically
As far as I know, there are several ways to achieve this

For now, we will treat this as a feature request

**Comment by julianxhokaxhiu** [comment]
Thanks a lot @yuk7 really appreciated! If there's anything I can do to help please let me know. Regarding the labelling makes total sense to me, feel free to work on this when you can and you feel to, no pressure or anything. It's a nice to have, but not urgent.

Cheers and best regards,
Julian

**Comment by sileshn** [comment]
@yuk7 @julianxhokaxhiu 

This can be closed. Not wsldl issue. I have fixed it in my ArchWSL2 repo. 

Sorry to both of you for the misleading information I provided.

**Comment by sileshn** [comment]
Btw, I got the information about the fix from the official [archwsl](https://gitlab.archlinux.org/archlinux/archlinux-wsl) repo.

As you may be aware, Archlinux is now officially supported by windows.
